### PR TITLE
Added Discount Filter and Showing discount code in attendee overview

### PR DESCRIPTION
**Pull request by LuD1161**
Solving issue [#3332](https://github.com/fossasia/open-event-orga-server/issues/3332)

### CODE REVIEW

File: app/helpers/ticketing.py [hunk 1]
```diff
@@ -48,11 +48,22 @@ def get_orders_of_user(user_id=None, upcoming_events=True):
...
+                   discount_code=None):
         if event_id:
             if status:
                 orders = Order.query.filter_by(event_id=event_id).filter_by(status=status) \
                     .filter(Order.user_id.isnot(None))
+            elif discount_code:
+                try:
+                    discount_id = DiscountCode.query.filter_by(event_id=event_id).filter_by(code=discount_code) \
+                        .filter(Order.user_id.isnot(None))[0].id
+                except Exception as e:
```
Comment (LuD1161) [inline app/helpers/ticketing.py]: Resolved it

Comment (LuD1161) [inline app/helpers/ticketing.py]: Removed

**Comment by mariobehling** [comment]
Please also resolve conflicts.

**Comment by LuD1161** [comment]
I don't get it , cause in the meantime I resolved one more issue #3339 and committed to my repo .
So , should I make a new pull request combining both the issues ?
Cause the new issue is what is making this conflict.

**Comment by mariobehling** [comment]
Only commit what is relevant to solve this issue. If you are solving another issue in the meantime, you can create a different branch and commit later from this branch.

**Comment by LuD1161** [comment]
@mariobehling 
Sorry , I am a bit new to this .
Will take care from now onwards.
However now what should I do ...
Should I make a new pull request , joining both the issues or can you suggest something better ?

**Comment by niranjan94** [comment]
@LuD1161 make two separate pull requests for each of the issues

**Comment by LuD1161** [comment]
@niranjan94
But I didn't know to create branch and went on committing to the same.
I think I will have to redo the changes :(

**Comment by niranjan94** [comment]
@LuD1161 please don't close and open PRs... Makes it very hard to track reviews/comments/conversation. Just force push to the same branch and the PR will automatically update.

**Pull request closed without merging**